tests/assignment-pattern/Makefile.in

```diff
@@ -0,0 +1,2 @@
+TOP_FILE := $(TEST_DIR)/top.sv
```

Comment:
This test will now run in Verilator as well, so it needs a *.cpp file (or at least an issue filed to add it later).

Reply:
This test is skipped for Verilator in CI: https://github.com/antmicro/verilator/blob/add-ci/.github/workflows/main.yml#L65